### DISCUSSION TITLE
Allow specifying Channel type with BufferLast parameter

### DISCUSSION THREAD

**Discussion by Danon5** [FirstGearGames/FishNet#581] [Feature Request]
The BufferLast parameter for the ObserversRpc is cool, but the packet can be dropped since it gets sent on the same channel. This is pretty annoying for things like unreliable position updates. It would be very helpful if it was possible to set a custom reliability type for the buffered RPC. All that needs to be added is a `Channel BufferLastChannel = Channel.Reliable` variable into the ObserverRpc attribute.

It should be reliable by default. I don't see why you would want to buffer an unreliable RPC just to have it be potentially dropped.

**Comment by maxkratt** [comment]
I also agree that it should be reliable by default, even if no option to change it is given.

**Comment by FirstGearGames** [comment]
I agree it would make sense for the bufferlast to always send on reliable.
That's a super easy change I'll get it into next v3 and v4.

**Comment by FirstGearGames** [comment]
Added in 4.0.7
